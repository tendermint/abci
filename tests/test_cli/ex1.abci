echo hello
info
commit
deliver_tx "abc"
info
commit
endblock 0
query "abc"
deliver_tx "def=xyz"
commit
query "def"
endblock 1
