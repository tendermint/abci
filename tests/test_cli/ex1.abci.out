> echo hello
-> data: hello
-> data.hex: 68656C6C6F

> info 
-> data: {"size":0}
-> data.hex: 7B2273697A65223A307D

> commit 

> deliver_tx "abc"

> info 
-> data: {"size":1}
-> data.hex: 7B2273697A65223A317D

> commit 
-> data: uü~„»×ˆíX–$ðlú‡EÑ
-> data.hex: 750502FC7E84BBD788ED589624F06CFA871845D1

> query "abc"
-> log: exists
-> height: 0
-> value: abc
-> value.hex: 616263

> deliver_tx "def=xyz"

> commit 
-> data: v9;Š.E†°iLbžËQ²†ïÕ
-> data.hex: 76393B8A182E450286B0694C629ECB51B286EFD5

> query "def"
-> log: exists
-> height: 0
-> value: xyz
-> value.hex: 78797A

